patches/components-content_settings-core-browser-content_settings_utils.cc.patch

```diff
@@ -1,19 +1,22 @@
 diff --git a/components/content_settings/core/browser/content_settings_utils.cc b/components/content_settings/core/browser/content_settings_utils.cc
-index 6b22ca5a8c671e5c761feb195537b22a2b513fef..ee575bd42f5c912d64df7aa69e70c5db92fe6300 100644
+index a66ca0ac3f01cc6a92c54bbaaef031b7e6414510..cd4cb9a38eb298218c7c36da83ff058c4b46a5b8 100644
 --- a/components/content_settings/core/browser/content_settings_utils.cc
 +++ b/components/content_settings/core/browser/content_settings_utils.cc
-@@ -156,6 +156,14 @@ void GetRendererContentSettingRules(const HostContentSettingsMap* map,
+@@ -143,11 +143,17 @@ void GetRendererContentSettingRules(const HostContentSettingsMap* map,
+ #endif
+   map->GetSettingsForOneType(ContentSettingsType::JAVASCRIPT,
+                              ResourceIdentifier(), &(rules->script_rules));
++  map->GetSettingsForOneType(ContentSettingsType::AUTOPLAY,
++                             ResourceIdentifier(), &(rules->autoplay_rules));
+   map->GetSettingsForOneType(ContentSettingsType::CLIENT_HINTS,
+                              ResourceIdentifier(),
                               &(rules->client_hints_rules));
-   map->GetSettingsForOneType(CONTENT_SETTINGS_TYPE_POPUPS, ResourceIdentifier(),
+   map->GetSettingsForOneType(ContentSettingsType::POPUPS, ResourceIdentifier(),
                               &(rules->popup_redirect_rules));
-+  map->GetSettingsForOneType(
-+      CONTENT_SETTINGS_TYPE_PLUGINS,
-+      "fingerprinting",
-+      &(rules->fingerprinting_rules));
-+  map->GetSettingsForOneType(
-+      CONTENT_SETTINGS_TYPE_PLUGINS,
-+      "braveShields",
-+      &(rules->brave_shields_rules));
++  map->GetSettingsForOneType(ContentSettingsType::PLUGINS, "fingerprinting",
```

Comment:
see https://github.com/brave/brave-core/pull/4374/files#r369669453
if you're making a non-trivial change to a patch, please convert it to a one line `define`